sdk/python/jobs/automl-standalone-jobs/automl-forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1429 @@
+    "            raw_data=partition_step.outputs.partitioned_data,\n",
+    "            automl_config=automl_config,\n",
+    "            max_concurrency_per_instance=max_concurrency_per_instance,\n",
+    "            prs_step_timeout=prs_step_timeout,\n",
+    "            instance_count=instance_count,\n",
+    "            enable_event_logger=enable_event_logger,\n",
+    "            retrain_failed_model=retrain_failed_model,\n",
+    "            compute_name=compute_name,\n",
+    "        )\n",
+    "    else:\n",
+    "        # 2. Model training step\n",
+    "        training_node = train_component_from_registry(\n",
+    "            raw_data=raw_data,\n",
+    "            automl_config=automl_config,\n",
+    "            max_concurrency_per_instance=max_concurrency_per_instance,\n",
+    "            prs_step_timeout=prs_step_timeout,\n",
+    "            instance_count=instance_count,\n",
+    "            enable_event_logger=enable_event_logger,\n",
+    "            retrain_failed_model=retrain_failed_model,\n",
+    "            compute_name=compute_name,\n",
+    "        )\n",
+    "\n",
+    "    # 3. Inferencing step\n",
+    "    inference_node = inference_component_from_registry(\n",
+    "        raw_data=inference_data,\n",
+    "        enable_event_logger=enable_event_logger,\n",
+    "        instance_count=instance_count,\n",
+    "        max_concurrency_per_instance=max_concurrency_per_instance,\n",
+    "        prs_step_timeout=prs_step_timeout,\n",
+    "        optional_train_metadata=training_node.outputs.run_output,\n",
+    "        forecast_mode=forecast_mode,\n",
+    "        forecast_step=forecast_step,\n",
+    "        compute_name=compute_name,\n",
+    "    )\n",
+    "\n",
+    "    # 4. Metrics calculation step\n",
+    "    compute_metrics_node = compute_metrics_component(\n",
+    "        task=\"tabular-forecasting\",\n",
+    "        prediction=inference_node.outputs.evaluation_data,\n",
+    "        ground_truth=inference_node.outputs.evaluation_data,\n",
+    "        evaluation_config=inference_node.outputs.evaluation_configs,\n",
+    "    )\n",
+    "    compute_metrics_node.compute = compute_name\n",
+    "\n",
+    "    # 5. Specify pipeline outputs\n",
+    "    return {\n",
+    "        \"output_files\": compute_metrics_node.outputs.evaluation_result,\n",
+    "        \"forecast_output\": inference_node.outputs.raw_predictions,\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 7. Kick Off Pipeline Runs\n",
+    "\n",
+    "Now that the pipline is defined, we will use it to kick off several runs. First, we will kick off an experiment which will evaluate the performance for the best AutoML model for each partition. Next, we will kick off the same pipeline which will only use the naive model for the same partitions. This will allow us to establish a baseline and compare performance results."
```

Comment:
This reads like we're kicking off an inference-only experiment to me. Maybe "...we will kick off an experiment which will train and evaluate...".